core/lib/src/trace.rs

```diff
@@ -224,6 +224,9 @@ pub mod prelude {
 pub enum LogLevel {
     /// Only shows errors and warnings: `"critical"`.
     Critical,
+    /// Shows errors, warnings, and some informational messages that are likely
+    /// to be relevant when troubleshooting (such as configuration).
```

Comment:
nit: looks like the other levels have docs that include the string name of that level in the config file, should that be included here?